ZENKO-3254: added two procedures to install and upgrade XDM (#1147)
scality/Zenko: docs/docsource/operation/CLI/generate_an_account_root_access_key.rst

> .. _Generate an Account Root Access Key:

Reviewer: ```suggestion
.. _Generate Account Root User Access Keys:
```

Reviewer 2: Is the point of this that when one generates an access key, one generates an access key and a secret key? If so, I'd argue for "Generate an Account Root Access Key Pair."

Reviewer 2: changed to: Generate Root User Account Access Keys

> Generate an Account Root Access Key

Reviewer: ```suggestion
Generate Account Root User Access Keys
```

===================================

> To generate an account key from the command line:

Reviewer: ```suggestion
To generate account root user access keys from the command line:
```

Reviewer 2: changed to "root user account access keys"
